### ISSUE TITLE
scripts/Makefile.config adds duplicate definitions for CONFIG_HAVE_L2CC

### ISSUE THREAD

**Issue by dandersonee** [atmelcorp/atmel-software-package#44]
scripts/Makefile.config contains code:

```
ifeq ($(CONFIG_HAVE_L2CC),y)
CFLAGS_DEFS += -DCONFIG_HAVE_L2CC
CONFIG_HAVE_L2CACHE=y
endif
ifeq ($(CONFIG_HAVE_SAIC),y)
CFLAGS_DEFS += -DCONFIG_HAVE_SAIC
endif
ifeq ($(CONFIG_USE_ROM_GALOIS_TABLE),y)
CFLAGS_DEFS += -DCONFIG_USE_ROM_GALOIS_TABLE
endif
ifeq ($(CONFIG_HAVE_L2CC),y)
CFLAGS_DEFS += -DCONFIG_HAVE_L2CC
endif
 
```

The second definition should be removed.

**Comment by loiclefort** [comment]
Thanks, will fix.

**Comment by loiclefort** [comment]
Fixed in release 2.9
